.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bug Report
+about: If things aren't working as expected.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Bug Report
+
+<!--
+Thanks for filing an issue! Before hitting the button, please answer these questions.
+Fill in as much of the template below as you can. If you leave out information, we can't help you as well.
+
+Note: Make sure to first check the prerequisites that can be found in the main README.md file!
+-->
+
+#### What did you do?
+
+<!-- A clear and concise description of the steps you took (or insert a code snippet). -->
+
+#### What did you expect to see?
+
+<!-- A clear and concise description of what you expected to happen (or insert a code snippet). -->
+
+#### What did you see instead? Under which circumstances?
+
+<!-- A clear and concise description of what you expected to happen (or insert a code snippet). -->
+
+#### Environment
+
+**Kubernetes cluster type:**
+
+<!-- The type of cluster used for testing/deployment, ex. "vanilla", "OpenShift" -->
+
+`$ Mention kubernetes-client/fabric8 version from pom.xml file`
+
+<!-- Verify pom.xml file and insert the java-operator-sdk version here. -->
+
+`$ java -version`
+
+<!-- Insert the output of `java -version` here -->
+
+`$ kubectl version`
```

Comment:
```suggestion
`Fabric8 Kubernetes Client Version`
```

Reply:
Asking user to provide kubectl version doesn't make sense in context of Kubernetes Java SDK? Maybe we should remove this section altogether? WDYT?